coalib/bearlib/languages/definitions/ApacheConf.py

```diff
@@ -0,0 +1,6 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class ApacheConf:
+    extensions = '.htaccess', 'apache.conf', 'apache2.conf'
```

Comment:
should we include filenames here? *..conf is going to conflict with many other formats.